### PR TITLE
Add request using relationships between tables

### PR DESCRIPTION
**Pull request by Grandi0z**
#### Change made : 
- Add `vets` table
- Add join tables `specializations` and `visits`
- Insert data
- Add queries using primary key & foreign key mechanisms for joining tables and relationships between tables

#### Note: Here are the results of added queries

1. 
req1

2. 
req2

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -26,3 +26,35 @@ ALTER TABLE animals ADD COLUMN  species_id INT;
...
+CREATE TABLE specializations (
+    species_id INT,
+    vets_id INT,
+    PRIMARY KEY(species_id, vets_id));
+
+CREATE TABLE visits (
+    animals_id INT,
+    vets_id INT,
+    date DATE,
+    PRIMARY KEY(animals_id, vets_id, date));
```
Comment (1GORDON) [inline schema.sql]: Great job @Grandi0z 🤝

- [ ] As stated in the project requirements, kindly ensure to add the `visit_date` column to the `visits` table so as to track the date of the visit.

File: data.sql [hunk 2]
```diff
@@ -40,3 +40,24 @@ UPDATE animals SET owner_id = CASE
   ELSE owner_id
 END;
```
Comment (1GORDON) [inline data.sql:42]: Great job @Grandi0z 🤝

- [ ] As stated in the project requirements, kindly add data for the `vets` table, `specialization` table, and `visits` table to the `data.sql` file.
- [ ] As stated in the project requirements, kindly create queries to answer questions in
transactions. Kindly add these queries to the `queries.sql` file.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Grandi0z ✋

Good job so far!

## Highlights!
- Created visits table ✔️ 
- Created specialization table ✔️
- Created vets table ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tugsem — APPROVED** [review]
## STATUS: APPROVED 👍🎯  
Hello @Grandi0z  👋 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⭐ 
✔️ Required changes implemented
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**